etl/steps/data/meadow/hmd/2022-11-03/hmd_lt.py

```diff
@@ -0,0 +1,322 @@
+import os
```

Comment:
The name of the step is not very informative, what about `life_tables` instead? I'd also suggest having a small docstring for the file (I think that's at the moment the only source of generic documentation for data steps).